Lesson10/Task15.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * 	Сделайте форму, которая спрашивает две даты в формате '2025-12-31'. Первую дату запишите в переменную $date1, а вторую в $date2. Сравните, какая из введенных дат больше. Выведите ее на экран.
+ */
+if (
+    !empty($_POST['date1']) 
+    && !empty($_POST['date2'])
+    && preg_match('#^\d{4}-\d{2}-\d{2}$#', $_POST['date1'])
+    && preg_match('#^\d{4}-\d{2}-\d{2}$#', $_POST['date2'])
```

Comment:
Can we check it without regular expression?

Reply:
Regular expression helps to follow the conditions of the task: "input:date" and "input:datetime-local" don't fit it. So we use regular expression to check date format.